### ISSUE TITLE
buildah cannot work with truncated containerID 

### ISSUE THREAD

**Issue by gouyang** [projectatomic/buildah#77]
```
$ sudo ./buildah containers
CONTAINER ID IMAGE ID     IMAGE NAME CONTAINER NAME
f7c575d24354 fd5230ef7b97 docker.io/library/redis:latest redis-working-container
$ sudo ./buildah mount f7c575d24354
ERRO[0000] error reading build container "f7c575d24354": error reading build container: container not known 
```

mount container name or the long containerID works.
```
$ sudo ./buildah mount redis-working-container
/var/lib/containers/storage/overlay2/241545a9bb5af26e0b60ba9af4af6900727618f011df2b13e2b0581f17231609/merged
$ sudo ./buildah containers -q
f7c575d2435448b5bb9567bcc33aef1480834f1d9b7ae5ed439dd54ee005ca79
$ sudo ./buildah mount f7c575d2435448b5bb9567bcc33aef1480834f1d9b7ae5ed439dd54ee005ca79
/var/lib/containers/storage/overlay2/241545a9bb5af26e0b60ba9af4af6900727618f011df2b13e2b0581f17231609/merged
```

**Comment by gouyang** [comment]
It looks all buildah cli could not work with the short containerID. Since `buildah containers` shows the short containerID, It should be supported.

```
sudo ./buildah containers
CONTAINER ID IMAGE ID     IMAGE NAME CONTAINER NAME
64a5b76698f9 60e4301d2634 docker.io/library/nginx:latest nginx-working-container
```

**Comment by nalind** [comment]
This _should_ just start working once we merge https://github.com/containers/storage/pull/6.

**Comment by rhatdan** [comment]
So this pull request was merged, now we need to update the vendor to buildah and we should be all set.

**Comment by nalind** [comment]
That PR introduced a couple of bugs, but I think https://github.com/containers/storage/pull/35 and https://github.com/containers/storage/pull/36 fix them.

**Comment by nalind** [comment]
Okay, this should be fixed by #79.

**Comment by nalind** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [(8) [Buildah] Add "dockah build-using-dockerfile" aka "buildah bud"](https://trello.com/c/AyBXqoRY/443-8-buildah-add-dockah-build-using-dockerfile-aka-buildah-bud)
